index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style/main.css">
+    <script src="./javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+
+    <header id="nav-container">
+        <div class="nav-socials-top">
+            <ul class="nav-socials">
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i> </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="fa-brands fa-twitter"></i>
+                    </a>
+                </li>               
+                <li>
+                    <a href="#">English</a>
+                </li>
+                <li>
+                    <a href="#"> My page</a>
+                </li>
+                <li>
+                    <a href="#">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <nav class="navbar">
+            <!-- <div class="inner-nav"> -->
+                <a class="logo-container" href="index.html">
+                    <img class="logo" src="./assets/icons/Drum-festival-logo.svg" alt="Drum Festival Logo">
+                </a>
+           
+                <ul class="navlinks-wrapper">
+                    <li>
+                        <a href="./html/about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#programs">Program</a>
+                    </li>
+                    <li>
+                        <a href="#Sponsors">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="#performing-artist">News</a>
+                    </li>
+                </ul>           
+            
+            <button type="button" class="nav-join-btn">
+                Join Drum Festival
+            </button>
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
+        </nav>
+    </header>
+
+    <!-- <div class="back-drop">
+        <div class="faded-backdrop"></div>
+        <div class="headline-bg-image"></div>      
+    </div> -->
```

Comment:
- [ ] **[OPTIONAL]**

- Let's keep our codebase clean by removing unused commented codes.

Reply:
Hi @zieeco , thanks for the code review. I am even more happy you are also a drummer,...nice one. 
I would like to connect with you more.